src/table/table.scss

```diff
@@ -36,10 +36,14 @@
 
   .iui-cell:not(.iui-slot) {
     @include iui-focus;
-    cursor: pointer;
+
+    &.iui-actionable {
+      cursor: pointer;
+    }
 
     > .iui-filter-button {
       margin-left: $iui-xs;
+      margin-right: $iui-xs;
```

Comment:
@FlyersPh9 Can you check what the value here should be?

0 (current):
![image](https://user-images.githubusercontent.com/9084735/121899482-790d5180-ccf2-11eb-846e-d607c996e5c9.png)

xs (4px):
![image](https://user-images.githubusercontent.com/9084735/121899540-84607d00-ccf2-11eb-873d-b56ec9a6f2d8.png)

s (8px)
![image](https://user-images.githubusercontent.com/9084735/121899603-92ae9900-ccf2-11eb-82b1-10501bc0e82e.png)

Reply:
My initial thought is to have the spacing between the button and the right edge of the column to be equal to either the red or orange spacing in the image below.
![Screen Shot 2021-06-14 at 11 45 53 AM](https://user-images.githubusercontent.com/849817/121920470-145bf200-cd06-11eb-97ee-8f35a3c68260.png)

I know the red gap is 16px, which is probably not our best use of space.  And the orange gap is probably too close to the edge of the column and a user might accidentally hover over the column resizer.  So I'd say let's go with the 8px gap.

Reply:
Changed to `iui-s`